examples/MNIST/README.md

```diff
@@ -15,7 +15,8 @@ Run the following command to train MNIST model
 CPU: `mlflow run . -P max_epochs=5`
 GPU: `mlflow run . -P max_epochs=5 -P gpus=2 -P accelerator=ddp`
 
-On the training completion, the MNIST model is stored as "model.pth" in current working directory.
+At the end of the training, MNIST model will be logged into mlflow as state dict and registers the mnist 
```

Comment:
@shrinath-suresh  Please consider adding the proper steps of running a server with sql backend and setting the tracking uri, prior to running the script.

Reply:
makes sense. I will update the README.md file to have sql backend as prerequisite.

Reply:
@ankan94 Ankan do we have the rst file ready for sql backend ?